### ISSUE TITLE
Lost pod "terminal" and "logs" menu options since 4.0 on windows

### ISSUE THREAD

**Issue by crochik** [lensapp/lens#2404]
This is a bug report with what worked for me. Hopefully will help others and also help fix it for a next release.

I have tried pretty much every release from the 4.X branch on my windows machine and could never got the pod "logs" and "terminal" icons to show again. I read somewhere that was because they became extensions in 4.0.

After uninstalling, removing every trace of lens that I could find on my machine, installing the latest 4.1.5 release and making no progress, I decided to look under the covers.... 

I quickly noticed:
![image](https://user-images.githubusercontent.com/10382865/112910444-fc86b180-90b8-11eb-9ec3-268f72d7b33e.png)

Turns out that I didn't have any files/folders under `C:\Users\{username}\AppData\Roaming\Lens\node_modules`

A little check under the folder where I installed Lens and I found "...\resources\extensions" ... The folder names don't match exactly what the errors message said but... quick copy of the folders to `C:\Users\{username}\AppData\Roaming\Lens\node_modules` and some renaming ....

from
![image](https://user-images.githubusercontent.com/10382865/112910623-5edfb200-90b9-11eb-88be-9a6ff252220c.png)
to
![image](https://user-images.githubusercontent.com/10382865/112910636-6606c000-90b9-11eb-83b6-780f729e07be.png)

and we are back in business...

**Comment by jim-docker** [comment]
Thanks for this, it should help us figure this out

**Comment by etlweather** [comment]
Any news on fix for this? I just found this project but then immediately ran into this problem. I am reading the workaround above but didn't seem to work out for me yet.

**Comment by Nokel81** [comment]
@etlweather Do you have anything under `%AppData%/Lens/Extensions`?

**Comment by etlweather** [comment]
@Nokel81 This folder does not exist.

![2021-06-18_123116](https://user-images.githubusercontent.com/3467506/122591613-2fde3a00-d031-11eb-9d32-c0fa5df4d427.png)

**Comment by etlweather** [comment]
Now I noticed now that at startup, it is trying to create symlinks from the install location (`c:\program files...`) to the `%appdata%` folder and it is getting permission denied because the `%appdata%` folder is on a remote server (redirected profiles in windows).

**Comment by Nokel81** [comment]
That would probably be the issue

**Comment by etlweather** [comment]
Yeah... looking for a way to tell Electron to store the app data somewhere else. I tried it on a non Domain computer with local files and it worked totally fine if I installed "just for me".

**Comment by Nokel81** [comment]
Maybe we could add support for some environment variables so that users like you could set up a different folder.

Because if you cannot write to `%AppData%\Lens` then other parts of Lens won't work either.

**Comment by etlweather** [comment]
Thank you and yes, that would be great, it could make the app also more portable if you had the program's binaries and `appdata` all into one place.

But to clarify, the problem isn't that I can't write in `%AppData%`, I definitely can and it does create files. The problem is that NodeJS is trying to create some `symlinks` which you can do on Windows but not on a network share and in my case `%AppData%` points to a network share (i.e. `\\server\user-app-data\username`)

**Comment by Nokel81** [comment]
> Thank you and yes, that would be great, it could make the app also more portable if you had the program's binaries and `appdata` all into one place.
> 
> But to clarify, the problem isn't that I can't write in `%AppData%`, I definitely can and it does create files. The problem is that NodeJS is trying to create some `symlinks` which you can do on Windows but not on a network share and in my case `%AppData%` points to a network share (i.e. `\\server\user-app-data\username`)
> 
> 

FYI @jakolehm

**Comment by jakolehm** [comment]
Lens 5 betas already bundle these extensions as real npm packages (tgz) -> no symlinks involved.
